plugins/mcreator-localization/help/default/entity/celebration_sound.md

```diff
@@ -0,0 +1,3 @@
+This sound will be played if the entity wins the raid.

```

Comment:
This does not explain the sound is only selectable when entity is of Raider type

Reply:
Fixed